Add Career History Spider

Deviad opened the pull request:
Career history spider keeps track of the political parties where a politician has served his country and of the corresponding years.
I think with this we could create a graph with the movements of every politician or something like that.

razvanpavel committed:
Merge pull request #29 from Owlree/develop

Added spider for Europarl

Deviad committed:
added spider to retrieve year and political party;

Owlree commented:
Hello, thanks for the pull request!

The following is an excerpt from the output of the new spider.
```
{
    "politician_name": "Mircea-Titus DOBRE",
    "url": "http://www.cdep.ro/pls/parlam/structura2015.mp?idm=97&cam=2&leg=2016",
    "political_party_name": "Formatiunea politica:"
},
{
    "politician_name": "Mara-Daniela CALISTA",
    "url": "http://www.cdep.ro/pls/parlam/structura2015.mp?idm=57&cam=2&leg=2016",
    "political_party_name": "Formatiunea politica:"
}
```
I can only assume the spider is not finished yet, and thus I cannot perform the merge. The output does not represent any form of career history. Also, the `political_party_name` field also grabs the wrong thing from the HTML.

The contribution document is WIP right now, but generally we would like enhancement pull requests that add new features (i.e. spiders) to only contain code that contributes to the project in a more relevant way (i.e. output data is usable at the time of the PR).

Those being said, I am going to close this PR for now.

Deviad commented:
Hello Roberta,
I revised my code, and fixed the issue.

https://github.com/code4romania/catalog-data-chambers/pull/31



On Mon, Jul 31, 2017 at 3:25 PM, Robert Badea <notifications@github.com>
wrote:

> Hello, thanks for the pull request!
>
> The following is an excerpt from the output of the new spider.
>
> {
>     "politician_name": "Mircea-Titus DOBRE",
>     "url": "http://www.cdep.ro/pls/parlam/structura2015.mp?idm=97&cam=2&leg=2016",
>     "political_party_name": "Formatiunea politica:"
> },
> {
>     "politician_name": "Mara-Daniela CALISTA",
>     "url": "http://www.cdep.ro/pls/parlam/structura2015.mp?idm=57&cam=2&leg=2016",
>     "political_party_name": "Formatiunea politica:"
> }
>
> I can only assume the spider is not finished yet, and thus I cannot
> perform the merge. The output does not represent any form of career
> history. Also, the political_party_name field also grabs the wrong thing
> from the HTML.
>
> The contribution document is WIP right now, but generally we would like
> enhancement pull requests that add new features (i.e. spiders) to only
> contain code that contributes to the project in a more relevant way (i.e.
> output data is usable at the time of the PR).
>
> Those being said, I am going to close this PR for now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/code4romania/catalog-data-chambers/pull/30#issuecomment-319052441>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACIpfEJelYnrooOyAJHOa-Qq6s7854nIks5sTce8gaJpZM4Oh4Oc>
> .
>

Deviad commented:
Hello Robert,
I revised my code and fixed the issue.

https://github.com/code4romania/catalog-data-chambers/pull/31

Thank you for highlighting that issue.

Regards,
Davide

On Mon, Jul 31, 2017 at 4:57 PM, Deviad <deviad@gmail.com> wrote:

> Hello Roberta,
> I revised my code, and fixed the issue.
>
> https://github.com/code4romania/catalog-data-chambers/pull/31
>
>
>
> On Mon, Jul 31, 2017 at 3:25 PM, Robert Badea <notifications@github.com>
> wrote:
>
>> Hello, thanks for the pull request!
>>
>> The following is an excerpt from the output of the new spider.
>>
>> {
>>     "politician_name": "Mircea-Titus DOBRE",
>>     "url": "http://www.cdep.ro/pls/parlam/structura2015.mp?idm=97&cam=2&leg=2016",
>>     "political_party_name": "Formatiunea politica:"
>> },
>> {
>>     "politician_name": "Mara-Daniela CALISTA",
>>     "url": "http://www.cdep.ro/pls/parlam/structura2015.mp?idm=57&cam=2&leg=2016",
>>     "political_party_name": "Formatiunea politica:"
>> }
>>
>> I can only assume the spider is not finished yet, and thus I cannot
>> perform the merge. The output does not represent any form of career
>> history. Also, the political_party_name field also grabs the wrong thing
>> from the HTML.
>>
>> The contribution document is WIP right now, but generally we would like
>> enhancement pull requests that add new features (i.e. spiders) to only
>> contain code that contributes to the project in a more relevant way (i.e.
>> output data is usable at the time of the PR).
>>
>> Those being said, I am going to close this PR for now.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/code4romania/catalog-data-chambers/pull/30#issuecomment-319052441>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ACIpfEJelYnrooOyAJHOa-Qq6s7854nIks5sTce8gaJpZM4Oh4Oc>
>> .
>>
>
>
